stuffer/s2n_stuffer_pem.c

```diff
@@ -139,16 +139,16 @@ int s2n_stuffer_private_key_from_pem(struct s2n_stuffer *pem, struct s2n_stuffer
         return S2N_SUCCESS;
     }
 
-    s2n_stuffer_reread(pem);
-    s2n_stuffer_reread(asn1);
+    POSIX_GUARD(s2n_stuffer_reread(pem));
+    POSIX_GUARD(s2n_stuffer_reread(asn1));
 
     /* By default, OpenSSL tools always generate both "EC PARAMETERS" and "EC PRIVATE
      * KEY" PEM objects in the keyfile. Skip the first "EC PARAMETERS" object so that we're
      * compatible with OpenSSL's default output, and since "EC PARAMETERS" is
      * only needed for non-standard curves that aren't currently supported.
      */
     if (s2n_stuffer_data_from_pem(pem, asn1, S2N_PEM_EC_PARAMETERS) != S2N_SUCCESS) {
-        s2n_stuffer_reread(pem);
+        POSIX_GUARD(s2n_stuffer_reread(pem));
     }
     s2n_stuffer_wipe(asn1);
```

Comment:
Might as well fix this one too, even if you can't enforce the MUST_USE. It's right there 😅 Alternatively you could switch it to reread like the others.
```suggestion
    }
    POSIX_GUARD(s2n_stuffer_wipe(asn1));
```